game/player.py

```diff
@@ -12,9 +12,7 @@ def color(self):
         return self._color
 
     def is_winner(self, board):
-        if board.game_over() and board.winner_color() == self._color:
-            return True
-        return False
+        return board.game_over() and board.winner_color() == self._color
```

Comment:
Just a very minor knit, but in general it is a good practice to append an `is_` when defining inquiring messages that return boolean values. So in this case `board.is_game_over()` is more explicit and leaves no doubts about what it does (on the other hand, `board.game_over()` in other context may mean that you are actually finishing the game)